buildenv/jenkins/triggerGrinder.groovy

```diff
@@ -1,38 +1,27 @@
 #!groovy
 
-def json_path = args.size() ? args[0] : "disabledTestParser/output.json."
+def json_path = args.size() ? args[0] : "disabledTestParser/output.json"
```

Comment:
Plz change it to `def json_path = "disabledTestParser/output.json"`

Reply:
Done.